src/components/cms/BotBuilder/BotWizard.js

```diff
@@ -104,11 +104,11 @@ const StepperCol = styled(Col)`
 `;
 
 const ContainerCol = styled(Col)`
-  position: fixed;
-  margin-left: 65%;
   height: 88%;
   margin-top: 10px;
-
+  position: sticky;
```

Comment:
Why added this? Is it required?

Reply:
To make the preview section fix to the screen. But when footer comes up it will scroll up.

Reply:
With previous do you mean: https://github.com/fossasia/susi.ai/pull/2466#discussion_r301667655